splunk/api_big_stats_dashboard.xml

```diff
@@ -124,7 +124,7 @@
                     <done>
                         <set token="tokTcCount">$result.count$</set>
                     </done>
-                    <query>index=bluebutton source="/var/log/pyapps/perf_mon.log*" host=$bbEnv$ | spath output=call_path path="message.path" | where call_path != "/health" | stats count</query>
+                    <query>index=bluebutton source="/var/log/pyapps/perf_mon.log*" host=$bbEnv$ | spath output=call_path path="message.path" | where call_path != "/health" AND call_path != "/health/external" AND call_path != "/health/external_v2" AND call_path != "/health/bfd" AND call_path != "/health/bfd_v2" AND call_path != "/health/sls" AND call_path != "/health/db" | stats count</query>

```

Comment:
@JFU-GIT This looks good. Was testing it out in Splunk also and counts are improved!

Just had one change that would handle a case where we add another /health endpoint in the future.

You can change `spath output=call_path path="message.path" | where call_path != "/health"`  to `search message.path!="/health*"` in the searches.

Example:

```suggestion
                    <query>index=bluebutton source="/var/log/pyapps/perf_mon.log*" host=$bbEnv$ | search message.path!="/health*" | stats count</query>

```

Reply:
> 

for some reason, the search did not carve out those health paths, here are the screen shots:
the API vs Total (minus health checks) using search pattern:
![image](https://user-images.githubusercontent.com/25412191/143918739-497f9622-7824-4ca9-9d29-177c2fa845ee.png)
the one with spath regex filters:
![image](https://user-images.githubusercontent.com/25412191/143918910-2e553515-209f-4d2b-9603-f92e988a750f.png)

Reply:
my bad, the search filter works as expected.